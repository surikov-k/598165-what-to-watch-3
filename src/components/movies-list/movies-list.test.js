import React from 'react';
import renderer from 'react-test-renderer';
import MoviesList from './movies-list.jsx';

const movies = [{
  id: 0,
  title: ``,
  genre: ``,
  release: 0,
  rating: 0,
  description: [``],
  director: [``],
  cast: [``],
  preview: ``,
  poster: ``,
  background: ``,
}];

describe(`<MoviesList/>`, () => {
  it(`renders correctly`, () => {
    const component = renderer.
      create(<MoviesList
        movies={movies}
        onMouseEnter={() => {}}
        onClick={() => {}}
      />).
      toJSON();
    expect(component).toMatchSnapshot();
  });
});
