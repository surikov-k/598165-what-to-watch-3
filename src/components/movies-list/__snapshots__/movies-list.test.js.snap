// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MoviesList/> renders correctly 1`] = `
Array [
  <div
    className="catalog__movies-list"
  >
    <article
      className="small-movie-card catalog__movies-card"
      onMouseEnter={[Function]}
    >
      <div
        className="small-movie-card__image"
        onClick={[Function]}
      >
        <img
          alt=""
          height="175"
          src=""
          width="280"
        />
      </div>
      <h3
        className="small-movie-card__title"
        onClick={[Function]}
      >
        <a
          className="small-movie-card__link"
          href=""
        >
          
        </a>
      </h3>
    </article>
  </div>,
  <div
    className="catalog__more"
  >
    <button
      className="catalog__button"
      type="button"
    >
      Show more
    </button>
  </div>,
]
`;
